stats/index.js

```diff
@@ -12,9 +12,8 @@ for (var j = 0; j < 100; ++j) {
   let stages = []
 
   for (var i = 1; i < 4; ++i) {
-    let utxos = Simulation.generateTxos(20 / i, min, max)
-    let txos = Simulation.generateTxos(80 / i, min, max / 3)
-
+    let utxos = Simulation.generateTxos(20 / i, min, max, {106: 0.9, 300: 0.1})
```

Comment:
@runn1ng perhaps its better to have the output sizes not represent averages,   but "profiles".   No one is going to have the same mixture of scripts as the "average".